packages/plugin-aws-elasticsearch/src/plugin.ts

```diff
@@ -0,0 +1,129 @@
+import assert = require('assert')
+import { resolve } from 'path'
+import retry = require('bluebird-retry')
+import Bluebird = require('bluebird')
```

Comment:
I'd recommend not using bluebird anymore as async/await support is widely available and performs well